src/fmu/dataio/_model/fields.py

```diff
@@ -556,6 +557,14 @@ class Context(BaseModel):
     See :class:`enums.FMUContext`."""
 
 
+class CaseContext(Context):
+    stage: Literal[enums.FMUContext.case] = Field(alias="stage")
+
+
+class IterationContext(Context):
+    stage: Literal[enums.FMUContext.iteration] = Field(alias="stage")
```

Comment:
The `alias` here could be removed as it is the same as the field name.
And probably it would be nice to use a default to avoid having to set this value 🙂 ?

```suggestion
    stage: Literal[enums.FMUContext.iteration] = Field(default=enums.FMUContext.iteration)
```